src/components/Checkout/KivaCardItem.vue

```diff
@@ -1,69 +1,83 @@
 <template>
-	<div class="basket-item-wrapper row">
-		<div class="hide-for-small-only medium-3 large-2 columns">
+	<div class="basket-item-wrapper tw-flex tw-flex-col md:tw-flex-row tw-pb-5">
+		<div class="tw-hidden md:tw-block tw-flex-none md:tw-mr-3 lg:tw-mr-4.5">
 			<div class="kiva-card-icon">
 				<!-- Print Kiva Card -->
 				<img
 					v-if="cardType == 'print'"
 					alt="print-kiva-card"
```

Comment:
Minor, these alts could just be `alt=""` since they seem decorative (the text next to them says the same words)